gobblin-runtime/src/main/java/org/apache/gobblin/runtime/embedded/EmbeddedGobblin.java

```diff
@@ -532,7 +539,7 @@ private void dumpJStackOnTimeout(String loc) {
     }
   }
 
-  private Configurable getSysConfig() {
+  public Configurable getSysConfig() {
```

Comment:
Is this public just for testing? We usually put the @VisibleForTesting annotation when access is relaxed just for the purpose of testing.

Reply:
Thanks for reminding me!